### ISSUE TITLE
What work is "duplicated"? 

### ISSUE THREAD

**Issue by ohmegasquared** [codeforboston/cliff-effects#553]
What are the concerns from managers having to do duplicate work? are they calculating this somewhere else? or what part is duplicate?

**Comment by knod** [comment]
There are some items on our form that they also fill out in the own forms. For example, all the income data. In some ways, we've mimicked the format of their existing forms.

Ideally, we would work with the API of their existing software and integrate it. The picture's a little more complicated than that, though, if I understand correctly. If I'm right, there are lots of different software solutions for these kinds of organizations and I'm not sure how we'd tackle integrating with different APIs, or even if most of those products have APIs that we'd have access to. I think the one our partner uses may have one, but we'd have to think about what our ultimate goal there would be.

Is it enough to integrate with just the software our partners use? Is it important to do so, even though it wouldn't accurately reflect our ability to integrate with other systems?

I think it's important to try and think of solutions to this, but haven't found one yet that makes sense in a bigger picture. Maybe the project would really end up trying to integrate with multiple small systems, at least as a transition (I don't think it'd be maintainable), because it's pretty unreasonable to expect the case managers to constantly do double entries.

**Comment by knod** [comment]
I had an idea for a possible solution!

Create a form that a facility can fill in that will dictate how their exported object properties fit our properties. That can be used to automatically import their data into our form!

I'm not sure how we can experiment with this, though. Kristin feels it's not a priority right now and folks there are currently too busy to make time for something that's not a priority, so we have no access to an example export of their data. We could create the very basic outline for it, but can't really implement it till we understand the kind of data we'd be getting. We'd also want data from other organizations. Folks from CfA also feel this isn't a priority at this time and we should wait till our partners have determined a list of current requirements before starting a new task. Still, worth keeping in mind!

**Comment by knod** [comment]
On hold till another phase of the project.
